tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/ConstSimplifierBase.scala

```diff
@@ -204,6 +205,11 @@ abstract class ConstSimplifierBase {
     // IF x THEN y ELSE y = y
     case OperEx(TlaControlOper.ifThenElse, _, thenEx, elseEx) if (thenEx == elseEx) => thenEx
 
+    // \E x \in S: FALSE = FALSE
+    case OperEx(TlaBoolOper.exists, _, _, ValEx(TlaBool(false))) => falseEx
+    // \A x \in S: TRUE = TRUE
```

Comment:
Isn't `\A x \in S: FALSE = FALSE` also a simplification required to propagate this and avoid the bug?

Reply:
Perhaps, one more expression to simplify then? I always have to recall the definition of conjunction and disjunction over the empty sets :)

Reply:
Yeah, I'll add some more equivalencies for the quantifiers.

Reply:
See #1347